cmd/ansible-operator/main.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
See the error faced in the sanity-test: 

```
Running golangci-lint
cmd/helm-operator/main.go:7: File is not `goimports`-ed (goimports)
	"github.com/operator-framework/operator-sdk/pkg/log/zap"
	logf "sigs.k8s.io/controller-runtime/pkg/log"
	log "github.com/sirupsen/logrus"
Makefile:89: recipe for target 'lint' failed
make: *** [lint] Error 1
```

It can be solved by running `make lint-fix`